src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,124 @@
+export const jsonToCol = (languages: string): any => {
+  let datas: any = JSON.parse(languages);
+  let dataArray: any = [];
+  let keys: any = [];
+  let columns: any = [];
+  let columnsValues: any = [];
+  let rows: any = [];
+  let fullRowData: any = [];
+  let columnTitle: any = [{ type: "text", title: "messageID", width: 150 }];
+
+  const setTitle = () => {
```

Comment:
stop embedding functions in to another functions

Reply:
What should I do then? any suggestion?

Reply:
yes write top level functions like `mapLangToSheetColumns()`, `mapLangToSheetData()`